setup-build-env/install_bpf_gcc.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -eu
+
+source $(cd $(dirname $0) && pwd)/../helpers.sh
+
+LOGFILE=build-gnu-bpf.log
+INSTALLDIR=$(pwd)/install_bpf_gcc/
```

Comment:
I have mixed filling about using `pwd` here. It is probably better to stick to `${GITHUB_WORKSPACE}` here so it is consistently at the same location and not dependent on the current working directory the workflow is running the action in.

Reply:
Sure.
Please notice that the `source $(cd $(dirname $0) && pwd)/../helpers.sh` is already in setup-build-env/install_clang.sh.

Reply:
Yes. This is fine. I meant this comment for INSTALL_DIR. The former is basically to build a path to the helper file in repo, regardless of the working directory.